packages/toolkit/src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -497,61 +487,32 @@ function copyProjectDependencies(buildCommand: CodeTransformMavenBuildCommand, m
     return [folderPath, folderName]
 }
 
-export async function zipCode(modulePath: string) {
+export async function zipCode() {
+    const modulePath = transformByQState.getProjectPath()
     throwIfCancelled()
     const zipStartTime = Date.now()
     const sourceFolder = modulePath
     const sourceFiles = getFilesRecursively(sourceFolder, false)
 
-    let mavenWrapperInstallFailed = false
     try {
-        installProjectDependencies('mvnw', modulePath)
+        installProjectDependencies()
     } catch (err) {
-        mavenWrapperInstallFailed = true
-    }
-
-    let mavenInstallFailed = false
-    if (mavenWrapperInstallFailed) {
-        try {
-            installProjectDependencies('mvn', modulePath)
-        } catch (err) {
-            mavenInstallFailed = true
-        }
-    }
-
-    const installFailed = mavenInstallFailed && mavenWrapperInstallFailed
-
-    if (installFailed) {
         void vscode.window.showErrorMessage(CodeWhispererConstants.installErrorMessage)
+        throw err
     }
 
     throwIfCancelled()
 
     let dependencyFolderInfo: string[] = []
-    let mavenWrapperCopyDepsFailed = false
     try {
-        dependencyFolderInfo = copyProjectDependencies('mvnw', modulePath)
+        dependencyFolderInfo = copyProjectDependencies()
     } catch (err) {
-        mavenWrapperCopyDepsFailed = true
-    }
-
-    let mavenCopyDepsFailed = false
-    if (mavenWrapperCopyDepsFailed) {
-        try {
-            dependencyFolderInfo = copyProjectDependencies('mvn', modulePath)
-        } catch (err) {
-            mavenCopyDepsFailed = true
-        }
-    }
-
-    const copyDependenciesFailed = mavenCopyDepsFailed && mavenWrapperCopyDepsFailed
-
-    if (copyDependenciesFailed) {
         void vscode.window.showErrorMessage(CodeWhispererConstants.dependencyErrorMessage)
+        throw err
     }
```

Comment:
In the two places above, we simply `throw err` because if either `Maven` shell command fails we simply stop the transformation process.